libs/charts/src/lib.rs

```diff
@@ -28,7 +28,7 @@
 //! All allocation-related data is stored in a global state in the [`data`] module. It features a
 //! [`Watcher`] type which, after [`start`]ing it, will monitor a directory for init and diff files.
 //!
-//! [`Json`]: ./base/trait.Json.html (The Json trait)
+//! [`Json`]: yew::format::Json (The Json trait)
```

Comment:
Yeah this kind of Rust-level linking (as opposed to html-level linking) is much better. Maybe rewrite the rest of them like that too, or write a TODO somewhere.

Reply:
Yes! I'll take care about this